src/storm/logic/TimeBoundType.h

```diff
@@ -41,7 +42,21 @@ class TimeBoundReference {
         return type == TimeBoundType::Reward;
     }
 
+    std::string const& getRewardModelName() const {
+        assert(isRewardBound());
+        return rewardName.get();
+    }
+
     std::string const& getRewardName() const {
+        return getRewardModelName();
+    }
+
+    bool hasRewardModelName() const {
+        assert(isRewardBound());
+        return static_cast<bool>(rewardName);
```

Comment:
This might be the more expected behaviour (I'm not sure?):
```suggestion
        return isRewardBound() && static_cast<bool>(rewardName);
```

Reply:
Changed it